internal/cmd/gencli/input.go

```diff
@@ -155,7 +155,6 @@ var inputStructs = map[string][]*cmdInfo{
 			Pkg:          "authtokens",
```

Comment:
Not in scope for this review... it would be nice to replace all these repeated strings (create, read, groups, Scope) with consts.  I just went throught this with my PR for adding ClaimsScopes to the oidc repo and after chasing a few typos, I replaced all the field names with consts and just eliminated future possible typo defects.

Reply:
Good idea for the future, yep.